### PR TITLE
feat(route): add Bangumi 成员关注动画榜

### PR DESCRIPTION
**Pull request by honue**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/bangumi/tv/followrank
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Documentation / 文档说明
  - [ ] EN / 英文文档
  - [x] CN / 中文文档
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/bangumi/tv/other/followrank.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const url = 'https://bgm.tv/anime';
+    const response = await got({
+        url,
+        method: 'get',
+        headers: {
+            'User-Agent': 'RSSHub (https://docs.rsshub.app/routes/anime#bangumi-fan-zu-ji-hua)',
+        },
```
Comment (TonyRL) [inline lib/v2/bangumi/tv/other/followrank.js:11]: The default user agent(a randomised chrome on mac) works. Is there any specific reasons to change it?

File: lib/v2/bangumi/tv/other/followrank.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            rankList.push({
+                title: $(this).children('a').attr('title'),
+                link: 'https://bgm.tv' + $(this).children('a').attr('href'),
+            });
+        });
+
+    const items = rankList.map((item) => ({
+        title: item.title,
+        link: item.link,
+    }));
```
Comment (TonyRL) [inline lib/v2/bangumi/tv/other/followrank.js:41]:
```suggestion
    const $ = cheerio.load(response.body);

    const items = $('#columnB > div:nth-child(4) > table > tbody')
        .find('tr')
        .toArray()
        .map((item) => {
            const aTag = $(item).children('td').next().find('a');
            return {
                title: aTag.html(),
                link: 'https://bgm.tv' + aTag.attr('href'),
            };
        })
        .concat(
            $('#chl_subitem > ul')
                .find('li')
                .toArray()
                .map((item) => ({
                    titile: $(item).children('a').attr('title'),
                    link: 'https://bgm.tv' + $(item).children('a').attr('href'),
                }))
        );

```

File: website/docs/routes/anime.mdx [hunk 3]
```diff
@@ -203,6 +203,10 @@ For more tags, please go to [Search torrent](https://bangumi.moe/search/index)
 
 <Route author="nczitzk" example="/bangumi/tv/user/blog/sai" path="/bangumi/tv/user/blog/:id" paramsDesc={['用户 id, 在用户页面地址栏查看']} radar="1"/>
 
+### 成员关注动画榜 {#bangumi-cheng-yuan-guan-zhu-dong-hua-bang}
+
+<Route author="honue" example="/bangumi/tv/followrank" path="/bangumi/tv/followrank" paramsDesc={[]} radar="1"/>
```
Comment (TonyRL) [inline website/docs/routes/anime.mdx:208]: Do not fill in `paramsDesc` if the path does not contain any path parameters
```suggestion
<Route author="honue" example="/bangumi/tv/followrank" path="/bangumi/tv/followrank"  radar="1"/>
```

File: lib/v2/bangumi/tv/other/followrank.js (hunk 1 above)
Comment (honue) [reply lib/v2/bangumi/tv/other/followrank.js:11]: yeah. the offical wanna know which program access their website. [link](https://github.com/bangumi/api/blob/master/docs-raw/user%20agent.md)

Comment (TonyRL) [reply lib/v2/bangumi/tv/other/followrank.js:11]: https://github.com/DIYgod/RSSHub/blob/05d53e01487ca8d0594e5d9c9f64e61ab8176473/lib/config.js#L5

Comment (honue) [reply lib/v2/bangumi/tv/other/followrank.js]: done. Thanks for your patience.

**Pull request merged**